.github/workflows/pre-main.yaml

```diff
@@ -355,6 +355,19 @@ jobs:
         env:
           IMAGE_TAG: ${TNF_IMAGE_TAG}
 
+      # Prepare collector to be used when running smoke tests
+      - name: Check out `Collector`
+        uses: actions/checkout@v4
+        with:
+          repository: shirmoran/collector
```

Comment:
Why test-network-function/collector is not good?

Reply:
it is temporary for testing purposes. once we'll merge the PR in the collector repo, it will be changed.